website/content/en/preview/upgrading/v1-migration.md

```diff
@@ -137,8 +137,20 @@ for [EKS Pod Identity ABAC policies](https://docs.aws.amazon.com/eks/latest/user
 
 * **metadataOptions could break workloads**: If you have workload pods that are not using `hostNetworking`, the updated default `metadataOptions` could cause your containers to break when you apply new EC2NodeClasses on v1.
 
-* **Support for native Ubuntu AMI selection is dropped**: If you are using Ubuntu and not using `amiSelectorTerms`, be aware that Karpenter v1 no longer natively supports Ubuntu. To continue using Ubuntu in v1, you can update `amiSelectorTerms` in your EC2NodeClasses to identify Ubuntu as the AMI you want to use. See [reimplement amiFamily](https://github.com/aws/karpenter-provider-aws/pull/6569) for an example. Once you have done that, you can leave the amiFamily as Ubuntu and proceed to upgrading to v1. This will result in the following transformation:
+* **Ubuntu AMIFamily Removed**:
+
+   Support for automatic AMI selection and UserData generation for Ubuntu has been dropped with Karpenter `v1.0.0`.
+   To continue using Ubuntu AMIs you will need to specify an AMI using `amiSelectorTerms`.
+
+   UserData generation can be achieved using the AL2 AMIFamily which, at the time of writing, has an identical UserData format.
```

Comment:
It's implied that at the time of writing it was identical. I think we can remove. It's unsupported anyway, so user will have to validate it themselves anyway.
```suggestion
   UserData generation can be achieved using the AL2 AMIFamily which has an identical UserData format.
```